src/_data/glossary.yml

```diff
@@ -1,3 +1,20 @@
+- term: "Analysis context"
+  short_description: |-
+    A representation of a set of code that should be analyzed using the same
+    sources of metadata.
```

Comment:
I think that "sources of metadata" might not be accurate, or the best terms. For one thing, I think the only source of "metadata" is a `package_config.json` file, so "source of metadata" would be fine.

And at that point you might just use "metadata" or something...

I was going to suggest the word "configuration" like "source configuration" or "package configuration file", but (a) I think most users are not really familiar with the package config file, and might mix it up with an _analysis options_ file, which at this point is quite distinct (and we took pains to separate them). So that's a little dicey. "Metadata" does nicely describe what distinguishes one set of sources from another, and doesn't _require_ users to know about the package config files.

Reply:
This PR is very droppable, no worries there. The real reason for it was that I wanted to emphasize the performance improvements and was only thinking about that in terms of analysis contexts, so thought I had to define them somewhere. But the way you worded it:

> the fact that using workspaces for large repositories will reduce the amount of memory used by the Dart language server and hence improve IDE performance

does a good job. Thanks for the useful information @srawlins and @bwilkerson! I will close this and change the reference in the new workspaces PR to _not_ mention analysis contexts specifically, just the potential performance benefits.